internal/pkg/deploy/cloudformation/stack/static_site.go

```diff
@@ -63,12 +65,62 @@ func NewStaticSite(conf *StaticSiteConfig) (*StaticSite, error) {
 
 // Template returns the CloudFormation template for the service parametrized for the environment.
 func (s *StaticSite) Template() (string, error) {
-	return "", nil
+	crs, err := convertCustomResources(s.rc.CustomResourcesURL)
+	if err != nil {
+		return "", err
+	}
+	addonsParams, err := s.addonsParameters()
+	if err != nil {
+		return "", err
+	}
+	addonsOutputs, err := s.addonsOutputs()
+	if err != nil {
+		return "", err
+	}
+
+	content, err := s.parser.ParseStaticSite(template.WorkloadOpts{
+		// Workload parameters.
+		AppName:            s.app,
+		EnvName:            s.env,
+		EnvVersion:         s.rc.EnvVersion,
+		SerializedManifest: string(s.rawManifest),
+		WorkloadName:       s.name,
+		WorkloadType:       manifestinfo.StaticSiteType,
+
+		// Additional options that are common between **all** workload templates.
+		AddonsExtraParams:   addonsParams,
+		NestedStack:         addonsOutputs,
+		PermissionsBoundary: s.permBound,
+
+		// Custom Resource Config.
+		CustomResources: crs,
+	})
+	if err != nil {
+		return "", err
+	}
+	return content.String(), nil
```

Comment:
Would there be any task def overrides before this?

Reply:
Task def override is only for ECS service. `static site` service type won't have an ECS service.